src/frontend/apps/lti_site/components/LiveInfoBar/index.spec.tsx

```diff
@@ -1,11 +1,32 @@
-import { getDefaultNormalizer, screen } from '@testing-library/react';
+import {
+  act,
+  getDefaultNormalizer,
+  screen,
+  waitFor,
+} from '@testing-library/react';
+import fetchMock from 'fetch-mock';
 import React from 'react';
 import { useParticipantsStore } from 'data/stores/useParticipantsStore/index';
 import render from 'utils/tests/render';
+import { report, useJwt } from 'lib-components';
 
 import { LiveInfoBar } from '.';
+import { wrapInVideo } from '../../utils/tests/wrapInVideo';
+import { videoMockFactory } from 'lib-components';
+import userEvent from '@testing-library/user-event';
+
+jest.mock('lib-components', () => ({
+  ...jest.requireActual('lib-components'),
+  report: jest.fn(),
+}));
 
 describe('<LiveInfoBar />', () => {
+  beforeEach(() => {
+    useJwt.setState({
+      jwt: 'json web token',
+    });
+  });
```

Comment:
```suggestion
  });
  
  afterEach(() => {
    fetchMock.reset();
    jest.resetAllMocks();
  });
```

This is important to reset the state of fetch-mock between each tests.
All jest mocks must be reset between each tests